### DISCUSSION TITLE
Feature - block email notification from a list of sender on forms

### DISCUSSION THREAD

**Discussion by mariohbrino** [statamic/cms#6102] [General]
Hi,

I'm looking for a solution to not send an email notification if there is a list of blocked email addresses for that specific form or any form, this would store the submitted form and avoid sending an email notification.

I still need the form to send notifications if the email address is not on the blocked list.

Thanks.

**Comment by edalzell** [comment]
You'll need to disable the default email sending and do it yourself in a listener.

**Comment by ryanmitchell** [comment]
You could store the emails in a global set, then use the [`FormSubmitted` event ](https://statamic.dev/extending/events#formsubmitted) to check the Global for an email match, and if there is one, you modify the event->submission->form->email() value... something like:

```
<?php

namespace App\Listeners;

use Statamic\Events\FormSubmitted;
use Statamic\Facades\GlobalSet;

class FormSubmittedListener
{
    public function handle(FormSubmitted $event)
    {
        $formHandle = $event->submission->form()->handle();

        if ($global = GlobalSet::find('your_global_handle')) {
            // some logic here
        
            // stop statamic sending emails by setting the emails to blank
            // this wont persist, it only changes for this request
            if ($shouldNotSendEmails) {
                $event->submission->form->email([]);   
            }
            
        }     
    }
}

```
